Userland/DevTools/HackStudio/Dialogs/Commit/GitCommitDialog.gml

```diff
@@ -0,0 +1,41 @@
+@GUI::Frame {
+    fixed_height: 260
+    fixed_width: 400
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        spacing: 3
+        margins: [4, 4, 4, 4]
+    }
+
+    @GUI::Label {
+        text: "Enter commit message:"
+        text_alignment: "CenterLeft"
+        fixed_height: 20
+    }
+
+    @GUI::TextEditor {
+        name: "message_editor"
+        placeholder: ""
+        fixed_height: 200
+    }
+
+    @GUI::Widget {
+        layout: @GUI::HorizontalBoxLayout
+        shrink_to_fit: true
+
+        @GUI::Widget {}
```

Comment:
There's no need to put curly braces after an element without properties.
```suggestion
        @GUI::Widget
```

Reply:
Wasn't aware of that, thank you.